askbot/jinja2/widgets/secondary_header.html

```diff
@@ -58,9 +51,6 @@
                             <i class="rover-icon rover-icon-house"></i>
                         </a>
                     </td>
-                    <td width="*" class="search-bar">{% include "widgets/search_bar.html" %}</td>
-                </tr>
-                <tr>
                     <td colspan="2">{% include "widgets/scope_nav.html" %}</td>
```

Comment:
This is a wide table cell (colspan="2"), that was paired with the previous row's 2 cells. I can't remember if colspan does anything if there's only a single table row, but I believe it's at least extraneous here.

Reply:
I think you're right. The appropriate update would likely be width="*", but I don't think it would change presentation